src/simplewallet/simplewallet.cpp

```diff
@@ -1681,7 +1681,13 @@ bool simple_wallet::open_wallet(const boost::program_options::variables_map& vm)
   {
     fail_msg_writer() << tr("failed to load wallet: ") << e.what();
     // only suggest removing cache if the password was actually correct
-    if (m_wallet && m_wallet->verify_password(password))
+    bool password_is_correct = false;
+    try
+    {
+      password_is_correct = m_wallet->verify_password(password);
```

Comment:
This seems to execute even when m_wallet is null.

Reply:
Thanks, fixed.